test/fileUploadParsers/OverseasTransferParserSpec.scala

```diff
@@ -58,68 +80,97 @@ class OverseasTransferParserSpec extends SpecBase with Matchers with MockitoSuga
-        ParserValidationError(2, 1, "memberDetails.error.lastName.required", "lastName"),
-        ParserValidationError(2, 3, "dob.error.incomplete", "dob",Seq("year")),
-        ParserValidationError(2, 2, "memberDetails.error.nino.invalid", "nino"),
-        ParserValidationError(2, 5, "chargeG.chargeDetails.qropsTransferDate.error.required.two", "qropsTransferDate",Seq("month","year")),
-        ParserValidationError(2,7, "amountTaxDue.error.invalid", "amountTaxDue")
-      ))
-    }
-
-    "return validation errors for charge amounts when present" in {
-      val result = parser.parse(startDate, invalidChargeAmountsCsvFile,UserAnswers())
-      result mustBe Left(Seq(
-        ParserValidationError(1, 6, "Enter the amount transferred into the QROPS for first last", "amountTransferred"),
-        ParserValidationError(2, 7, "amountTaxDue.error.invalid", "amountTaxDue")
+        ParserValidationError(0, 0, HeaderInvalidOrFileIsEmpty)
       ))
     }
 
+        "return validation error for empty file" in {
+          val result = parser.parse(startDate, Nil,UserAnswers())
+          result mustBe Left(Seq(
+            ParserValidationError(0, 0, HeaderInvalidOrFileIsEmpty)
+          ))
+        }
+
+            "return validation error for not enough fields" in {
+              val GivingNotEnoughFields = CsvLineSplitter.split(
+                s"""$header
+                            one,two"""
+              )
+              val result = parser.parse(startDate, GivingNotEnoughFields,UserAnswers())
+              result mustBe Left(Seq(
+                ParserValidationError(1, 0, NotEnoughFields)
+              ))
+            }
+
+                "return validation errors for member details when present" in {
+                  val GivingInvalidMemberDetails = CsvLineSplitter.split(
+                    s"""$header
+                            ,last,AB123456C,01/04/2000,123123,01/04/2020,1.00,2.00
+                            Joe,,123456C,01/04/2000,123123,01/04/2020,1.00,2.00"""
+                  )
+
+                  val result = parser.parse(startDate, GivingInvalidMemberDetails,UserAnswers())
+                  result mustBe Left(Seq(
+                    ParserValidationError(1, 0, "memberDetails.error.firstName.required", "firstName"),
+                    ParserValidationError(2, 1, "memberDetails.error.lastName.required", "lastName"),
+                    ParserValidationError(2, 2, "memberDetails.error.nino.invalid", "nino")
+                  ))
+                }
+
+                   "return validation errors for charge details when present, including missing year and missing month" in {
+                     val GivingInvalidChargeDetails = CsvLineSplitter.split(
+                       s"""$header
+                            first,last,AB123456C,01,123123,01/04,1.00,2.00
```

Comment:
Formatting is a bit awry here. Could you format it?

Reply:
done